ricecooker/config.py

```diff
@@ -15,8 +15,11 @@
 STAGE = False
 LOGGER = logging.getLogger()
 
-# Domain and file store location for uploading to production server
-DOMAIN = os.getenv('CONTENTWORKSHOP_URL', "https://contentworkshop.learningequality.org")
+# Domain and file store location for uploading to production Studio server
+DOMAIN_ENV = os.getenv('STUDIO_URL', None)
+if DOMAIN_ENV is None:  # check old ENV varable for backward compatibility
+    DOMAIN_ENV = os.getenv('CONTENTWORKSHOP_URL', None)
+DOMAIN = DOMAIN_ENV if DOMAIN_ENV else "https://studio.learningequality.org"
```

Comment:
Might want to keep contentworkshop since cloudflare prevents uploads of over 500MB

Reply:
@jayoshih 500MB should be OK, but since you were asking about a possible workaround in case we want to support 500MB+ single-file uploads, the ricecooker user could set SUSHIBAR_URL env var to "https://contentworkshop.learningequality.org" to avoid the cloudflare middleman.